app/components/info.tsx

```diff
@@ -0,0 +1,60 @@
+interface WeAreBlockProps {
```

Comment:
lets also rename the file to be we-are-block and move both components under a homepage folder
/components/home-page